src/Tags.zig

```diff
@@ -96,24 +100,101 @@ pub fn findTags(self: *Tags, fname: []const u8) anyerror!void {
     gop.value_ptr.* = {};
 
     const mapped = a: {
-        var file = try std.fs.cwd().openFile(fname, .{});
-        defer file.close();
+        if (OS_TAG == .windows) {
+            const w = std.os.windows;
+
+            const stupid_alloc_fname = try self.allocator.dupeZ(u8, fname);
+            defer self.allocator.free(stupid_alloc_fname);
+
+            const dwAttrib = windowsApi.GetFileAttributesA(stupid_alloc_fname);
+
+            if ((dwAttrib & w.FILE_ATTRIBUTE_DIRECTORY) != 0) {
+                return error.NotFile;
+            }
+
+            const hfile = windowsApi.CreateFileA(stupid_alloc_fname, w.GENERIC_READ, // dwDesiredAccess
+                0, // dwShareMode
```

Comment:
the comments were from the example I found, idk if you want to keep those?

Reply:
Seems fine